src/io/flutter/settings/FlutterSettings.java

```diff
@@ -26,6 +30,9 @@
   private static final String syncAndroidLibrariesKey = "io.flutter.syncAndroidLibraries";
   private static final String disableTrackWidgetCreationKey = "io.flutter.disableTrackWidgetCreation";
   private static final String useFlutterLogView = "io.flutter.useLogView";
+  // The Dart plugin uses this registry key to keep bazel users from getting their settings overridden on projects that include a
+  // pubspec.yaml.
+  private static final String ignorePubspecRegistryKey = "dart.projects.without.pubspec";
```

Comment:
Rename to something like `DART_PROJECTS_WITHOUT_PUBSPEC_KEY`, the current name makes it sound like you are going to ignore the value stored in `dart.projects.without.pubspec`.

Reply:
Good idea, done.